### PR TITLE
Switch to catkin tools, quad namespace other minor improvements

### PR DESCRIPTION
**Pull request by jcnorby**

### CODE REVIEW

File: external/osqp-eigen/LICENSE [hunk 1]
```diff
@@ -145,7 +145,7 @@ choice, if you do both of the following:
 
   The Free Software Foundation may publish revised and/or new versions
 of the GNU Lesser General Public License from time to time. Such new
-versions will be similar in spirit to the present version, but may
```
Comment (justinyim) [inline external/osqp-eigen/LICENSE:148]: Revert mistaken find-replace in package's license file

**Review by yanhaoy — APPROVED** [review]

File: quad_logger/scripts/processLog.m [hunk 2]
```diff
@@ -10,24 +10,24 @@ function processLog(varargin)
...
 
 % Check that this is the right current directory otherwise paths won't work
-if ~endsWith(pwd, 'spirit-software/spirit_logger/scripts')
-    error('This script must be run from spirit-software/spirit_logger/scripts/');
+if ~endsWith(pwd, 'quad-software/quad_logger/scripts')
+    error('This script must be run from quad-software/quad_logger/scripts/');
 end
 
 % Import URDF
-spirit40 = importrobot('../../spirit_simulator/spirit_description/urdf/spirit.urdf');
```
Comment (justinyim) [inline quad_logger/scripts/processLog.m:24]: Actually spirit?

File: quad_logger/scripts/processLog.m [hunk 3]
```diff
@@ -10,24 +10,24 @@ function processLog(varargin)
...
+quad40 = importrobot('../../quad_simulator/spirit_description/urdf/spirit.urdf');
 %figure
-% homeConfig = homeConfiguration(spirit40);
```
Comment (justinyim) [inline quad_logger/scripts/processLog.m:26]: Actually Spirit? (same for a few lines below)

File: quad_simulator/README.md [hunk 4]
```diff
@@ -291,7 +291,7 @@ After the conversion:
 ```
 2. Make sure to move the SDF file to the [vision60 SDF folder](vision60_description/sdf) and rename it to vision60.sdf.
 
-## Converting Spirit's URDF to SDF
```
Comment (justinyim) [inline quad_simulator/README.md:294]: Actually Spirit

File: quad_simulator/README.md [hunk 5]
```diff
@@ -55,7 +55,7 @@ for Vision60, or
 ```
 $ roslaunch gazebo_scripts spirit_gazebo.launch
 ```
-for Spirit.
```
Comment (justinyim) [inline quad_simulator/README.md:58]: Actually Spirit?

File: quad_simulator/spirit_description/sdf_mesh/model.config [hunk 6]
```diff
@@ -34,6 +34,6 @@ SOFTWARE.
     <email>nflowers@andrew.cmu.edu</email>
   </author>
   <description>
-    Model of the Spirit body with .stl meshes for visualization
+    Model of the Quad body with .stl meshes for visualization
   </description>
```
Comment (justinyim) [inline quad_simulator/spirit_description/sdf_mesh/model.config:38]: Actually Spirit

**Review by justinyim — CHANGES_REQUESTED** [review]
Some spirit -> quad mistaken renames

**Review by justinyim — APPROVED** [review]
Fixed erroneous spirit -> quad replacements

**Pull request merged**